controllers/statistics.js

```diff
@@ -0,0 +1,168 @@
+import models from '../models';
+import timeWasted from '../helpers/readingTime';
+
+const { User, article, statistic } = models;
+
+const userAttrib = ['username', 'image', 'bio'];
+const articleAttrib = ['title', 'description', 'slug', 'tagList'];
+
+const statistics = {
+  createStats: async (req, res) => {
+    const { id } = req.user;
+    const { timeSpentReading } = req.query;
+    const convertToMins = timeSpentReading / 60;
+    const { slug } = req.query;
+    const totalReadTime = timeWasted.nonReadTime(convertToMins);
+    try {
+      const articles = await article.findOne({
+        where: { slug },
+        attributes: ['id', 'title', 'body', 'slug'],
+        include: {
+          model: User,
+          attributes: userAttrib
+        }
+      });
+      if (totalReadTime <= 0.3333) {
+        return res.status(409).send({
+          status: res.statusCode,
+          message: 'User has just scrolled through',
```

Comment:
The error message should be more descriptive. `Why can't we accept the provided value?` Something like `The value provided is below the minimum`

Reply:
@avpaul I believe this message is descriptive enough owing to the fact that it aims at the article that the user has read or scrolled through